.github/workflows/auto-fork.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 The Knative Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Auto-Fork
+
+on:
+  schedule:
+  - cron: '0 10 * * 1-5' # 3am Pacific on weekdays
+
+  workflow_dispatch:
+
+jobs:
+
+  forky:
+    name: Auto-Fork
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false # Keep running if one leg fails.
+      matrix:
+        org:
+        - knative-sandbox
+        - knative
+
+    steps:
+
+    - name: Set up Go 1.15
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.x
+
+    - name: Install Dependencies
+      run: go get knative.dev/test-infra/buoy@master
+
+    - name: Install gh
+      run: |
+        sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-key C99B11DEB97541F0
+        sudo apt-add-repository https://cli.github.com/packages
+        sudo apt update
+        sudo apt install gh
+
+    - name: Check out code onto GOPATH
+      uses: actions/checkout@v2
+
+    - name: Check Repo
+      run: GITHUB_TOKEN=${{ PERSONAL_ACCESS_TOKEN }} ./hack/auto-fork.sh ${{ matrix.org }}
+
+    - name: Post failure notice to Slack
+      uses: rtCamp/action-slack-notify@v2.1.0
+      if: ${{ failure() }}
+      env:
+        SLACK_ICON: http://github.com/knative-automation.png?size=48
+        SLACK_USERNAME: knative-automation
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        SLACK_CHANNEL: 'productivity'
+        SLACK_COLOR: '#8E1600'
+        MSG_MINIMAL: 'true'
+        SLACK_TITLE: Auto-fork ${{ matrix.org }} failed.
```

Comment:
matrix.org?

Reply:
nevermind, I see it now

Reply:
https://github.com/knative-sandbox/knobots/pull/28/files#diff-8882dc6c91e8aa14dc9d6827e459711aabec7149d4c3a66354a81fd0d6f387acR32

pretty sure that is how it works